### ISSUE TITLE
[Question/Suggestion] Generation

### ISSUE THREAD

**Issue by Jophire** [MightyPirates/BedrockOres#83]
I was incredibly happy to see this mod updated to 1.19.2, but missed the older only 1 per chunk generation style.

As such I have spent some time learning how datapacks work to switch the generation over to a structure based style and have it mostly working. I am now only trying to figure out how to control the size of the bedrockore:bedrock_ore vein itself. If I want larger veins, I can just spawn multiple copies of the feature, but smaller ones are not currently possible. With modded ores for example I am looking at some where I would only want 1 block to spawn in the structure. 

TLDR: Can you make bedrockores:bedrock_ore spawn as a single block and change the generator to include a size variable.
`
{

  "type": "bedrockores:bedrock_ore",
  "config": {
    "size":10, --New variable to designate how many blocks to try and convert.
    "ore": {"Name": "minecraft:coal_ore"},
    "amount": {
      "type": "minecraft:uniform",
      "value": {
        "min_inclusive": 300,
        "max_inclusive": 500
      }
    }
  }
}`

**Comment by fnuecke** [comment]
I've fixed the height config of the built-in feature being broken, so you could now presumably spawn it with more controlled size (e.g. 1x1x1).  Not sure if that helps.  However, you might be interested in [this twitter thread](https://twitter.com/TTFTCUTS/status/1692236450924798372), which contains some info about custom spawning the ores.
